taxonomy-manager-rest-server/src/main/resources/db/procedures/skos_concept/R__get_concepts_by_partial_label.sql

```diff
@@ -0,0 +1,12 @@
+DROP FUNCTION IF EXISTS get_concepts_by_partial_label;
+CREATE OR REPLACE FUNCTION get_concepts_by_partial_label(_label text) RETURNS SETOF skos_concept AS
+$$
+BEGIN
+    RETURN QUERY
+        SELECT concept.*
+        FROM skos_concept concept
+        WHERE EXISTS(SELECT 1
+                     FROM jsonb_each_text(concept.preferred_label) AS preferred_label
+                     WHERE preferred_label.value::text ~ ('^' || _label));
```

Comment:
When we're doing a search it'll be for a specific language, so we need to only interrogate the values associated with that language key (and also the hidden labels, too).